AdyenCard/Components/Card/CardComponentExtensions.swift

```diff
@@ -27,12 +27,12 @@ extension CardComponent {
         return validator.isValid(key)
     }
     
-    private func getEncryptedCard(publicKey: String) throws -> CardEncryptor.EncryptedCard {
-        let card = CardEncryptor.Card(number: numberItem.value,
-                                      securityCode: securityCodeItem.value,
-                                      expiryMonth: expiryDateItem.value[0...1],
-                                      expiryYear: "20" + expiryDateItem.value[2...3])
-        return try CardEncryptor.encryptedCard(for: card, publicKey: publicKey)
+    private func getEncryptedCard(publicKey: String) throws -> EncryptedCard {
+        let card = UnencryptedCard(number: numberItem.value,
```

Comment:
`Unencrypted` prefix is unnecessary, just `Card` and `EncryptedCard` were just fine.

card is unencrypted be default :-D.